bCNC/Pendant.py

```diff
@@ -32,7 +32,7 @@
 	Image = None
 
 try:
-	import BaseHTTPServer as HTTPServer
+		import BaseHTTPServer as HTTPServer
```

Comment:
This indentation seems to be redundant to me...

Reply:
You right, i will fix it